kufar/books/service.go

```diff
@@ -23,9 +39,23 @@ func ReadAll(d Database) func(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+var secrets map[Credentials]BookResponse
+
 //Update ...
 func Update(d Database) func(w http.ResponseWriter, r *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		// r.SetBasicAuth(cred.User, cred.Password)
```

Comment:
Let's leave it as is for now. After we will create a valid auth scheme, will talk about proper middleware for auth :)